cluster/gce/util.sh

```diff
@@ -189,13 +189,26 @@ function detect-master () {
 }
 
 # Ensure that we have a password created for validating to the master.  Will
-# read from $HOME/.kubernetres_auth if available.
+# read from the kubernetes auth-file for the current context if available.
+#
+# Assumed vars
+#   KUBE_ROOT
 #
 # Vars set:
 #   KUBE_USER
 #   KUBE_PASSWORD
 function get-password {
-  local file="$HOME/.kubernetes_auth"
+  local get_auth_path='''
+import yaml, sys
+cfg = yaml.load(sys.stdin)
+try:
+  current = cfg["current-context"]
+  user = cfg["contexts"][current]["user"]
+  print cfg["users"][user]["auth-path"]
+except KeyError:
+  pass
+'''
+  local file=$("${KUBE_ROOT}/cluster/kubectl.sh" config view --global | python -c "${get_auth_path}")
```

Comment:
I would love to express my displeasure at inline python for this solution, but it's the most readable inline python in our codebase so far.

Something something need a YAML splitter for bash blah blah rant rant except this should be in go anyways LGTM.